spec/services/collators/legal_aid_collator_spec.rb

```diff
@@ -6,25 +6,27 @@ module Collators
     let(:disposable_income_summary) { assessment.applicant_disposable_income_summary }
 
     describe ".call" do
-      subject(:collator) { described_class.call(disposable_income_summary.legal_aid_outgoings) }
+      subject(:collator) { described_class.call(outgoings) }
 
       context "when there are no legal_aid outgoings" do
+        let(:outgoings) { [] }
+
         it "leaves the monthly maintenance field on the disposable income summary as zero" do
           expect(collator).to be_zero
         end
       end
 
       context "when there are legal_aid outgoings" do
-        before do
+        let(:outgoings) do
           # payments every 28 days which equals 112.08 per calendar month
-          create :legal_aid_outgoing, disposable_income_summary:, payment_date: 2.days.ago, amount: 103.46
-          create :legal_aid_outgoing, disposable_income_summary:, payment_date: 30.days.ago, amount: 103.46
-          create :legal_aid_outgoing, disposable_income_summary:, payment_date: 58.days.ago, amount: 103.46
+          [build(:legal_aid_outgoing, payment_date: 2.days.ago, amount: 103.46),
+           build(:legal_aid_outgoing, payment_date: 30.days.ago, amount: 103.46),
+           build(:legal_aid_outgoing, payment_date: 58.days.ago, amount: 103.46)]
 
           # childcare payments should be ignored
```

Comment:
@starswan Not sure why removing childcare_outgoing is not impacting monthly equivalent calculation 🤔

Reply:
It's all to do with parameter passing. Creating extra outgoings in these tests is a distraction - the code called gross_income_summary.legal_aid_outgoings *very* early on - so the 'extras' weren't that useful. In fact the *test* calls it, thus making the extras really confusing